src/main/java/com/ibm/cloud/sdk/core/service/BaseService.java

```diff
@@ -240,6 +240,20 @@ public static String constructServiceURL(
     return formattedUrl;
   }
 
+  /**
+   * @deprecated
+   * Deprecated in favor of `constructServiceUrl`.
+   * This naming convention is consistent with `setServiceUrl`.
```

Comment:
```suggestion
   * @deprecated use constructServiceUrl() instead.
```
Let's just keep the message simple like other deprecated tags.